sentry-rails/lib/sentry/rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sentry
   module Rails
-    VERSION = "4.1.6"
+    VERSION = "4.2.0"
```

Comment:
can you revert all the version changes? we have a different release procedure here 🙂